DocumentService/Controllers/DocumentsController.cs

```diff
@@ -146,13 +169,15 @@ public IActionResult DeleteDocumentById(Guid id, string userName)
                 if (isDeleted)
                 {
                     return new JsonResult(isDeleted);
```

Comment:
Same thing here too, return Ok instead of new JsonResult

Reply:
Same thing as the above comment.